velox/docs/develop/memory.rst

```diff
@@ -901,4 +901,4 @@ check the system memory usage periodically. Whenever the system memory usage
 exceeds a certain threshold, it tries to free up memory from Velox by shrinking
 the file cache (*AsyncDataCache::shrink*), and returns the freed cache memory
 back to the OS. This way we can automatically shrink the file cache in response
-to the transient spiky memory usage from non-Velox components in a query system.
\ No newline at end of file
+to the transient spiky memory usage from non-Velox components in a query system.
```

Comment:
@lingbin Can't tell what changed here. Would you clarify?

Reply:
```
\ No newline at end of file
```
"vim" will enforce a newline character at the end of the file by default.

At present, it seems that clang-format will ensure that there is a newline character at the end of our source file(`h|cpp files`), but there is no check for documents (`.rst` files). 
@mbasmanova  Should I revert this modification?

Reply:
I see. That's fine. Thank you for clarification.

It would be nice to enhance make format-fix/check to take care of this.